wandb/sdk/wandb_run.py

```diff
@@ -451,7 +452,30 @@ def _init(
                 sweep_config, user="sweep", _allow_val_change=True
             )
 
-        if (
+        if self._settings.launch and (
+            os.environ.get("WANDB_CONFIG") is not None
+            or os.environ.get("WANDB_ARTIFACTS") is not None
+        ):
+            if os.environ.get("WANDB_CONFIG") is not None:
+                try:
+                    new_config = ast.literal_eval(os.environ.get("WANDB_CONFIG", "{}"))
```

Comment:
Why are we using `ast.literal_eval` instead of a formal serialization like JSON or YAML?

Reply:
Oops, I'll fix that